packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Customize the Okta URL domain
+excerpt: Learn how to add a custom domain name to your Okta organization.
+layout: Guides
+---
+
+This guide teaches you how to customize the Okta URL domain which allows you to create a seamless branded experience for your users.
+
+---
+
+**Learning outcomes**
+
+Customize the Okta URL domain.
+
+**What you need**
+
+* A domain that you own, for example, `example.com`
+* A subdomain that you want to use, for example, `login.example.com`
+* A valid TLS certificate (PEM-encoded) for your subdomain
+* A 2048-bit private key (PEM-encoded)
+
+---
+
+## Overview
+
+You can customize your Okta organization by replacing the Okta domain name with your own domain name. This allows you to create a seamless branded experience for your users so that all URLs look like your application.
+
+Okta orgs host pages on subdomains such as `example.okta.com`. Using the custom URL domain feature aliases your Okta organization's domain name to another subdomain that you own, like `login.example.com`.
+
+For example, you use Okta as a user store for your apps, but you don't want your users to know that the app uses Okta behind the scenes. You can create a [CNAME record](https://en.wikipedia.org/wiki/CNAME_record) for the Okta domain, allowing you to alias `login.example.com` to `example.okta.com`.
+
+> **Note:** You must first customize the Okta URL domain if you also want to customize the Okta-hosted [sign-in page](/docs/guides/style-the-widget/style-okta-hosted/) or [error pages](/docs/guides/custom-error-pages/).
+
+Okta serves pages on your custom domain over HTTPS. To set up this feature, you need to provide a TLS certificate that is valid for your domain. See [Validate your TLS certificate](#validate-your-tls-certificate).
+
+### Caveats
+
+* Most custom domains are configured using one of your [subdomains](https://en.wikipedia.org/wiki/Subdomain). However, there are no restrictions on using any other domain such as your root domain, if that fits your use case for your sign-in page taking over your main website.
+
+* Okta currently only supports 2048-bit keys for the private key that you upload. However, your certificate chain can use keys of any size.
+
+* If your org has configured any SAML or WS-Fed integrated applications, review the SAML or WS-Fed SSO set-up instructions. If you want your customers to see the new custom domain rather than the Okta org domain, update those SAML or WS-Fed Service Provider integrations to use the new custom URL in the metadata.
```

Comment:
It seems weird to say this, but not actually link to the instructions the sentence is referencing. Can you provide those thinks?